Wspólne podkatalogi: wpa_supplicant-0.3.8.orig/doc i wpa_supplicant-0.3.8/doc
diff -u wpa_supplicant-0.3.8.orig/Makefile wpa_supplicant-0.3.8/Makefile
--- wpa_supplicant-0.3.8.orig/Makefile	2005-02-14 02:55:47.000000000 +0100
+++ wpa_supplicant-0.3.8/Makefile	2005-04-07 21:55:05.803706440 +0200
@@ -32,8 +32,10 @@
 	echo CONFIG_WIRELESS_EXTENSION=y >> .config
 
 install: all
-	mkdir -p $(DESTDIR)/usr/local/sbin/
-	for i in $(ALL); do cp $$i $(DESTDIR)/usr/local/sbin/$$i; done
+	mkdir -p $(DESTDIR)/etc
+	cp wpa_supplicant.conf $(DESTDIR)/etc
+	mkdir -p $(DESTDIR)/usr/sbin/
+	for i in $(ALL); do cp $$i $(DESTDIR)/usr/sbin/$$i; done
 
 OBJS = config.o \
 	eloop.o common.o md5.o \
